paper/paper.Rmd

```diff
@@ -0,0 +1,171 @@
+---
+title: "Static Code Analysis for R"
+date: "`r Sys.Date()`"
+tags: ["R", "linter", "tidyverse"]
+authors:
```

Comment:
I have added only a preliminary list of authors. I am not sure about what should either the details (authors can do this themselves) or the order here.